.github/workflows/review-release-notes.yml

```diff
@@ -0,0 +1,37 @@
+name: Review Release Notes
+on: pull_request
+
+jobs:
+  release_notes_review:
+    runs-on: ubuntu-latest
+    if: github.repository == 'demisto/content' && github.event.pull_request.head.repo.fork == false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v2.0.0
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install Python Dependencies
+        run: python -m pip install --upgrade pip
+      - name: Install Demisto SDK
+        run: pip install demisto-sdk
+      - name: Review Release Notes
+        run: |
+          overall_exit_code=0
+          file_exit_code=0
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            echo ""
```

Comment:
whats the empty echo for ?

Reply:
visual new line, it does not do well with \n for some reason